checkClock/app/helpers/dashboard_helper.rb

```diff
@@ -0,0 +1,2 @@
+module DashboardHelper
+end
```

Comment:
If you occupy it, delete it.

Reply:
Yes, we did not occupy that file, it has already been deleted in a new commit